### DISCUSSION TITLE
What the best thing to do with the beets’ Non-Album directory?

### DISCUSSION THREAD

**Discussion by nodecentral** [beetbox/beets#4608] [Q&A]
Having working my way through a large mp3 folder of random tracks in downloads, I now have a folder called `Non-Album` in my music ..

What’s the recommended approach I take next ? 

Do I try and group them where I can to make (incomplete) albums ? Or..
Do I make them singles tracks in their assigned artist folder ? Or..
Should I wrap them all up as a compilation?

**Comment by sampsyo** [comment]
Hello! Any particular reason why you don't like the "Non-Album" folder? Is it just that *everything* in your library is a singleton?

If so, I would just reconfigure my paths to get rid of that folder and put everything in the root:
https://beets.readthedocs.io/en/stable/reference/config.html#path-format-configuration

**Reply by nodecentral** [reply]
Hi @sampsyo

I must admit, I’m not a fan of it, it just seems like a dumping ground (which I guess it is :-)

I’d like to be able to associate and group them into actual albums, or perhaps assign them against their associated artist in maybe a folder called `Singles` - something that makes them seem more part of the beets structure/layout..

**Reply by sampsyo** [reply]
Certainly! A path format like `$artist/Singles/$title` could help accomplish that.

**Reply by nodecentral** [reply]
Hi @sampsyo , am I right in saying I have update the config.yml and then run the import them again against the `Non-Album` folder now in the music directory ?

**Reply by sampsyo** [reply]
No; this FAQ describes an easier way to move files to match a new `paths:` configuration:
https://beets.readthedocs.io/en/stable/faq.html#rename-my-files-according-to-a-new-path-format-configuration
